source/AndroidXTargets.cshtml

```diff
@@ -48,7 +48,7 @@
       <AndroidXSkipAndroidXMigration>true</AndroidXSkipAndroidXMigration>
     </AndroidAarLibrary>
       } else {
-    <AndroidJavaLibrary Include="$(MSBuildThisFileDirectory)..\..\jar\@(art.MavenGroupId).@(art.MavenArtifactId).jar">
+    <AndroidJavaLibrary Include="$(MSBuildThisFileDirectory)..\..\jar\@(art.MavenGroupId).@(art.MavenArtifactId).jar" Condition=" '$(AndroidApplication)' == 'true' ">
```

Comment:
Should I add the same check for `@(AndroidAarLibrary)` a few lines above?

To my knowledge `@(AndroidAarLibrary)` doesn't actually _do_ anything in a class library, but maybe just to keep the logic the same?